### PR TITLE
feat: Add utils for creating app context to run cli scripts

### PR DESCRIPTION
**Pull request by andrewi-wd**

### CODE REVIEW

File: scripts/utils/cliContextUtils.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import type { RequestContext } from '@lokalise/fastify-extras'
+import { generateMonotonicUuid } from '@lokalise/id-utils'
+import type { CommonLogger } from '@lokalise/node-core'
+import type z from 'zod'
+import { type AppInstance, getApp } from '../../src/app.js'
+
+export const getArgs = () => {
+  return process.argv.reduce((args: Record<string, unknown>, arg) => {
```
Comment (kibertoad) [inline scripts/utils/cliContextUtils.ts:8]: is it possible to simplify this code by using https://nodejs.org/api/util.html#utilparseargsconfig?

File: scripts/utils/cliContextUtils.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    } else if (arg.startsWith('-')) {
+      for (const flag of arg.slice(1)) {
+        args[flag] = true
+      }
+    }
+    return args
+  }, {})
+}
+
+export const createCliContext = async <Arguments>(
```
Comment (kibertoad) [inline scripts/utils/cliContextUtils.ts:23]: can you also add sample script using this, and also document the usage in the readme?

File: scripts/utils/cliContextUtils.ts (hunk 1 above)
Comment (andrewi-wd) [reply scripts/utils/cliContextUtils.ts]: Good catch! Added.

File: scripts/utils/cliContextUtils.ts (hunk 2 above)
Comment (andrewi-wd) [reply scripts/utils/cliContextUtils.ts:18]: Added both

**Review by kibertoad — APPROVED** [review]
Awesome job, thank you so much!

**Pull request merged**